File: backend/src/main/java/com/satc/integrador/ai/studyPlan/dto/ExercicioGramaticaOrdemGetDto.java
package com.satc.integrador.ai.studyPlan.dto;

import java.util.List;

public class ExercicioGramaticaOrdemGetDto {
    private Integer id;
    private Integer idOrdemExercicio;
    private Integer idPlanoEstudo;
    private String fraseCompleta;
    private List<String> ordemCorreta;
    private List<String> ordemAleatoria;
}

File: android/app/src/main/java/com/satc/integrador_ai/data/model/request/ExercicioGramaticaOrdemGetDto.kt
package com.satc.integrador_ai.data.model.request

import com.satc.integrador_ai.storage.TipoExercicio

class ExercicioGramaticaOrdemGetDto (
    val id: Int ?= null,
    val idOrdemExercicio: Int ?= null,
    val idPlanoEstudo: Int ?= null,
    val fraseCompleta: String ?= null,
    val opcaoCorreta: MutableList<String> ?= null,
    val ordemAleatoria: MutableList<String> ?= null
) : TipoExercicio {
    override fun getTipoExercicio(): String {
        return "GramaticaOrdem"
    }
}